backend/config/staffusers.json

```diff
@@ -0,0 +1,21 @@
+{
+  "readonly": [
+    "alexander.steel@gsa.gov"
+  ],
+  "helpdesk": [
+    "philip.dominguez@gsa.gov",
+    "robert.novak@gsa.gov",
+    "analyn.delossantos@gsa.gov",
+    "hassandeme.mamasambo@gsa.gov",
+    "iman.ali@gsa.gov",
```

Comment:
We can take `iman.ali@gsa.gov` out of the list

Reply:
@sambodeme Thank you - it would also make sense to add the new folks in this list as well.

Reply:
Only folks with access to helpdesk for now.